app/assets/javascripts/components/ReportFilter.jsx

```diff
@@ -4,8 +4,10 @@ class ReportFilter extends React.Component {
     const view_level = props.view_level || 'Genus';
     this.background_model = props.background_model || 'N/A';
     this.report_title = props.report_title || 'Not Set';
-    const default_nt_zscore_threshold = `0-${this.props.highest_tax_counts.highest_nt_zscore}`;
-    const default_nt_rpm_threshold = `0-${this.props.highest_tax_counts.highest_nt_rpm}`;
+    this.highest_nt_zscore = this.props.highest_tax_counts.highest_nt_zscore || 100;
+    this.highest_nt_rpm = this.props.highest_tax_counts.highest_nt_rpm || 100;
```

Comment:
Valid points, @cdebourcy    @cyrielo @eggerdesigns and I had a chat about filter ranges, and we decided that 1) the default range should be the min ... max from the report data  2) if min=max, the sliders will be turned off in UI.   This will address the issue that prompted this pull request, namely that a javascript exception was being thrown by the slider UI component when min=max.